src/fides/api/schemas/system.py

```diff
@@ -19,6 +19,10 @@ class PrivacyDeclarationResponse(PrivacyDeclaration):
     )
     cookies: Optional[List[Cookies]] = []
 
+    legal_basis_for_processing_override: Optional[str] = Field(
+        description="Global overrides for this purpose's legal basis for processing if applicable.  Defaults to the legal_basis_for_processing otherwise."
+    )
```

Comment:
This is the property I added to include this value in the System response.  As noted elsewhere, it differs by name from the hybrid property on the Privacy Declaration due to async issues.